lld/test/ELF/riscv64-reloc-got32_pcrel-overflow.s

```diff
@@ -0,0 +1,11 @@
+// REQUIRES: riscv
```

Comment:
Similar to the aarch64 patch, this negative test should be merged into `lld/test/ELF/riscv64-reloc-got32_pcrel.s`

I think we use `_` or `-` consistently. If there is a `-`, just use `-` for `_` in `got32_pcrel` as well